<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VignetteRecette</class>
 <widget class="QWidget" name="VignetteRecette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLabel{
	font:16px;
	color:black;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,20,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="50,50,0">
     <item>
      <widget class="QPushButton" name="modifButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">text-align: center;
font : bold;</string>
       </property>
       <property name="text">
        <string>Editer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSuppr">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">text-align: center;
font : bold;</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelNom">
     <property name="text">
      <string>Gateau au chocolat</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelImage">
     <property name="styleSheet">
      <string notr="true">QLabel#labelImage

{
	border-image: url(:/Images/Images/gateauChocolat.jpg) 0 0 0 0 stretch stretch;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pr√©paration :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDureePrep">
       <property name="text">
        <string>30 min</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
